pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneStarter.java

```diff
@@ -149,7 +149,7 @@ void start() throws Exception {
         if (!onlyBroker) {
             // Start LocalBookKeeper
             bkEnsemble = new LocalBookkeeperEnsemble(numOfBk, zkPort, bkPort, zkDir, bkDir, wipeData);
-            bkEnsemble.start();
+            bkEnsemble.startStandalone();
```

Comment:
No, I left that calling `start()` starts the "low-memory" bookie, while `startStandalone()` is what we use for the `bin/pulsar standalone` command.